R/validate.R

```diff
@@ -90,3 +90,44 @@
     )
   }
 }
+
+#' Validate inputs use to generate retrospective data
```

Comment:
line break between title and description

Reply:
This was removed